.env.development

```diff
@@ -2,7 +2,8 @@ VITE_TRACTION_BASE_URL=http://traction
 VITE_TRACTION_GRAPHQL_URL=http://traction/graphql
 VITE_PRINTMYBARCODE_BASE_URL=http://printmybarcode
 VITE_SAMPLEEXTRACTION_BASE_URL=http://sampleextraction
-VITE_LABWHERE_BASE_URL=http://labwhere
+VITE_LABWHERE_BASE_URL=http://localhost:3003
```

Comment:
I think it is better to do this in .env.development.local

Reply:
There is an e2e test to check for volume exhaustion when the entered location barcode matches the destroyed location barcode, as specified in the corresponding .env file. This test will fail during CI if the correct configuration is not set. Is there are any better approaches to handle this scenario?

Reply:
Ah ok. That is a problem. How does it work for all of our other api calls. We don't set traction url to a real one. It sounds like the intercept is not being handled correctly maybe?

Reply:
Here, we are simply comparing the entered barcode with the value of the environment variable read within the method. I think this is different from other cases where we can intercept commands to mock network requests and responses, eliminating the need for the real ones. I tried various approaches, such as using Cypress config to define a value, but it didn't work